erizo/buildProject.sh

```diff
@@ -3,23 +3,17 @@
 set -e
 
 BIN_DIR="build"
-OBJ_DIR="CMakeFiles"
 
 buildAll() {
   if [ -d $BIN_DIR ]; then
-    cd $BIN_DIR
-    for d in */ ; do
-      echo "Building $d - $*"
-      cd $d
-      make $*
-      cd ..
+    for entry in "$BIN_DIR"/*
```

Comment:
Is there anything in the previous version that is not working? I would remove these changes otherwise

Reply:
no just a cleaner syntax i think